components/l10n/common/locale_constants.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_L10N_COMMON_LOCALE_CONSTANTS_H_
+#define BRAVE_COMPONENTS_L10N_COMMON_LOCALE_CONSTANTS_H_
+
+namespace brave_l10n {
+
+constexpr char kDefaultLocale[] = "en_US";
```

Comment:
why are we hardcoding this here?

Reply:
See https://github.com/brave/brave-core/pull/15312#discussion_r990503893